src/constant/CustomerDealerEnum.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package constant;
+
+/**
+ *
+ * @author Haseeb
+ */
+public enum CustomerDealerEnum {
+    Select_TYPE, Dealer, Customer;
```

Comment:
May be we don't write Select_Type here.
and make it in Capital words.

Reply:
aggred